### PR TITLE
Added support links to exception and toolbar

### PR DESCRIPTION
**Pull request by peterrehm**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #17311 |
| License | MIT |
| Doc PR | - |

Exception Page:
bildschirmfoto 2016-01-10 um 17 33 09

Toolbar:
bildschirmfoto 2016-01-10 um 17 35 09

I am not yet sure if the link at the top of the exception is visible enough.

### CODE REVIEW

File: src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception.html.twig [hunk 1]
```diff
@@ -1,5 +1,10 @@
 <div class="block-exception">
     <div class="block-exception-detected clear-fix">
+        <div class="support">
+            <span>
```
Comment (dosten) [inline src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception.html.twig]: Why you need this <span>?

Comment (peterrehm) [inline src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception.html.twig]: To reuse the CSS definitions from the Log section.

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/config.html.twig [hunk 2]
```diff
@@ -109,6 +109,14 @@
                         {% endif %}
                     </span>
                 </div>
+                <div class="sf-toolbar-info-piece">
+                    <b>Support</b>
```
Comment (javiereguiluz) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/config.html.twig]: `Support` looks a bit redundant here. What about changing it by the following:

```html
 <b>Help</b>
 <span>
     <a href="http://symfony.com/support">
         Symfony Support
     </a>
 </span>
```

Comment (peterrehm) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/config.html.twig]: Updated accordingly.

**Comment by TomasVotruba** [comment]
If the "Symfony support" should be try link (not sure, can be a heading, based on screenshot), I'd add action verb.

"Find help at Symfony support"

**Comment by peterrehm** [comment]
@TomasVotruba What do you mean with "try link"? Somehow the link should be more visible.

**Comment by TomasVotruba** [comment]
@peterrehm Sorry, I meant "the link". The point was better visibility, yes. Stackoverflow icon might help as well.

**Comment by peterrehm** [comment]
@javiereguiluz What are your oppinions on that?

**Comment by javiereguiluz** [comment]
The idea would be to redesign the exception page in the near term to match the new Symfon style. So for now, I like the current proposal.

**Comment by peterrehm** [comment]
@javiereguiluz Do you want for the current one the top link more visible or are you okay with it as it is?

**Comment by WouterJ** [comment]
I agree with @TomasVotruba the label should include an action instead of just a subject. I like "Need support?".

File: src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception.html.twig (hunk 1 above)
Comment (WouterJ) [inline src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception.html.twig]: That can also be done when using `<div class="support"><a ...></div>`, by changing `.sf-reset .support span` to `.sf-reset .support a`

**Comment by peterrehm** [comment]
Updated:

bildschirmfoto 2016-01-21 um 20 13 24

Status: Needs Review

**Comment by fabpot** [comment]
The link color should probably be just black as the contrast between grey and green seems very low.

**Comment by peterrehm** [comment]
@fabpot Updated accordingly.

**Comment by fabpot** [comment]
Thank you @peterrehm.

**Pull request closed without merging**